src/web/mjs/connectors/futabanet.mjs

```diff
@@ -20,20 +20,33 @@ export default class futabanet extends SpeedBinb {
     }
 
     async _getMangas() {
-        //https://futabanet.jp/list/monster/works?page=1
-        let msg = 'Manga list is not yet implimented for this website, please copy and paste the URL containing the chapters directly from your browser into HakuNeko.';
-        throw new Error(msg);
+        let request = new Request(this.url + '/list/monster/serial', this.requestOptions);
+        let pages = await this.fetchDOM(request, 'li.m-pager__last a');
+        pages = Number( new URL(pages[0].href).searchParams.get('page') );
+
+        let data;
+        let mangas = [];
+        for (let page = 0; page <= pages; page++) {
+            request = new Request(this.url + '/list/monster/serial' + '?' + 'page' + '=' + page);
```

Comment:
May be wrapped in `new URL()`
Also combine constant strings `'a?b='` instead of `'a' + '?' + 'b' + '='`